inc/class-image-editor-imagick.php

```diff
@@ -64,10 +64,23 @@ public function load() {
 		$this->remote_filename = $this->file;
 		$this->file = $temp_filename;
 
-		$result = parent::load();
```

Comment:
Do know all the effects of removing the parent load. we should double check this.

Reply:
So I've had another good look at this to see if there is a why to reuse the parent method. The reason why it's only working when I instantiate a new object is because of the s3 logic file path logic just above. Basically the parent load() method can't load the object because it doesn't know where it is, and it doesn't know this becuase it is an unusal location...an s3 path. The custom code in the sub load class is what provides the path context and once it has that I can instantiate an object and go from there with it. I've also had a look at the wp parent load() method and it isn't doing anything I'm not doing in my one so I think this is good still to use. But open to discuss!

Reply:
👍